CMakeLists.txt

```diff
@@ -78,7 +78,6 @@ packageProject(
   INCLUDE_DIR ${PROJECT_SOURCE_DIR}/include
   INCLUDE_DESTINATION include/${PROJECT_NAME}-${PROJECT_VERSION}
   VERSION_HEADER "${VERSION_HEADER_LOCATION}"
-  COMPATIBILITY ExactVersion
   COMPATIBILITY SameMajorVersion
```

Comment:
with this  you static assert fails if an older version ist installed on staged!

Reply:
Yes, that is the intended consequence. The test-cases are only relevant during development and are not intended to be used for other versions than the current development branch.

Reply:
so we should go back to theExactVersion?

Reply:
Ah maybe I misunderstood the question. I think for most projects that `SameMajorVersion` is fine as usually an API should not contain breaking changes between minor releases.